0019-glossary/0019-glossary.md

```diff
@@ -236,6 +239,11 @@ See [IL-RFC 9](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-p
 ## Source (transfer, ledger, amount)
 The transfer/ledger/amount directly adjacent to the sender.
 
+## Streaming Payment
+An ongoing payment where small amounts of money are sent over time to pay for some ongoing service or rent. For instance, a user might pay for a streaming video with a streaming payment. Or a tenant may pay rent as a streaming payment.
+
+The amount sent in each iteration and the interval between iterations are inversely proportional. When the interval is long, e.g. a month, this would usually be called a subscription or recurring payment. The term "streaming payment" is commonly reserved for cases where the interval is short, from one second up to about a day.
```

Comment:
A more specific meaning of the term "streaming payments" in the context of http-ilp is the practice of streaming one single http resource in exchange for more than one Interledger payment.